src/System.AppContext/pkg/System.AppContext.pkgproj

```diff
@@ -9,12 +9,13 @@
     <ProjectReference Include="..\ref\System.AppContext.csproj">
       <SupportedFramework>netcore50;dnxcore50</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.AppContext.builds" />
-
-    <!-- Remove when resolving https://github.com/dotnet/corefx/issues/5471 -->
+    <ProjectReference Include="..\src\redist\System.AppContext.depproj">
```

Comment:
Yes. They must be passed to the computer so they **must** be in the reference package.